interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
 
 ### DataFactory
 
 **Methods:**
 
-- `.iri(String iri)` returns a new instance of IRI.
-- `.blankNode()` returns a new instance of BlankNode.
+- `.sym(String iri)` returns a new instance of IRI.
+- `.bNode()` returns a new instance of BlankNode.
```

Comment:
As it only reflects a name and not any architecture or functionality decision I don't see it too important. Following thoughts: 

1) Unfortunately is the term NamedNode not mentioned neither in [RDF1.1](http://www.w3.org/TR/rdf11-concepts/) nor in [TURTLE](http://www.w3.org/TR/turtle/). Than again because this low level API is focused mostly on experts, they should find their way around anyway.

2) I like to understand if a NamedNode simply is defined as disjoint to BlankNode (like in [RDF1.1](http://www.w3.org/TR/rdf11-concepts/))? In this case would the Literal be also a NamedNode?

On a purely practical level personally I feel that `.iri` is easier to grasp and work with in the daily business.